angularjs/angular.d.ts

```diff
@@ -651,6 +651,12 @@ declare namespace angular {
         [name: string]: any;
     }
 
+    interface IFilterOrderByObject {
+        value: any;
+        type: any;
+        index: any;
+    }
```

Comment:
`value` is a return value of one of the predicates (including the 'dummy predicate', see the docs). Different predicates can return values of different types. Also predicates can be expressions (strings), not functions. So I don't think it's possible to think of something better than `any` here.